src/InfoViz/Native/HistogramSelector/index.js

```diff
@@ -203,7 +226,7 @@ function histogramSelector(publicAPI, model) {
       }
       const mungedData = fieldNames.map(name => {
         const d = model.provider.getField(name);
-        if (!d.selectedGen) {
+        if (typeof d.selectedGen === 'undefined') {
```

Comment:
I think you should be able to write that condition like (d.selectedGen === undefined)